website/community/2-resources.mdx

```diff
@@ -66,6 +66,7 @@ See the <a href={require('@docusaurus/useBaseUrl').default('showcase')}>showcase
 - [docusaurus-plugin-typedoc](https://github.com/tgreyuk/typedoc-plugin-markdown/tree/master/packages/docusaurus-plugin-typedoc) - A Docusaurus plugin to build documentation with [TypeDoc](https://typedoc.org/)
 - [docusaurus-openapi-docs](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs) - A Docusaurus plugin and theme for generating interactive OpenAPI docs
 - [docusaurus-post-generator](https://github.com/moojing/docusaurus-post-generator) - A command line tool for user to add a blog/doc file quickly by command like `yarn gen-post new [template] [post_name]`.
+- [docusaurus-scheduler](https://github.com/alvarolorentedev/docusaurus-scheduler) - A docusarus compatible github action to schedule blog article releases based on a defined date in front-mater of md & mdx files.
```

Comment:
```suggestion
- [docusaurus-scheduler](https://github.com/alvarolorentedev/docusaurus-scheduler) - A Docusaurus compatible GitHub action to schedule blog article releases based on a defined date in front-mater of md & mdx files.
```

Reply:
Sure don't merge it then 🙂.

What it does is simple it uses a temporal folder and moves them to the blog folder of docusaurus in the correct date.